//* err: todo!
fn main() {
    let x = 2;
    x
}
